frontend/src/console/components/InstanceSettings/InstanceSettings.tsx

```diff
@@ -66,6 +69,24 @@ export class InstanceSettingsComponent extends React.PureComponent<
         publicContactEmail: this.props.activeInstance!.data!.publicContactEmail
       });
     }
+    // if current state is empty and pre-filling
```

Comment:
Maybe we could reword this to something along `Set an initial state or restore empty values`?